internal/webserver/server.go

```diff
@@ -126,6 +126,10 @@ func (webserver *WebServer) StartHTTPServer(errChannel chan error) {
 	webserver.LoggingClient.Info(fmt.Sprintf("Starting HTTP Server on port :%d", webserver.Config.Service.Port))
 	go func() {
 		p := fmt.Sprintf(":%d", webserver.Config.Service.Port)
-		errChannel <- http.ListenAndServe(p, http.TimeoutHandler(webserver.router, time.Millisecond*time.Duration(webserver.Config.Service.Timeout), "Request timed out"))
+		if serviceTimeout, err := time.ParseDuration(webserver.Config.Service.Timeout); err != nil {
+			errChannel <- err
```

Comment:
Do we need to give context to the error message, i.e. which setting failed to parse?

Reply:
yea, I'll add more context to it